projects/angular/src/accordion/accordion-panel.ts

```diff
@@ -52,12 +61,23 @@ export class ClrAccordionPanel implements OnInit, OnChanges {
   private _panelIndex: number;
 
   constructor(
+    @Optional()
+    @SkipSelf()
+    public parent: ClrAccordionPanel,
```

Comment:
Why is this `public`? I think it should either be `protected` or `private`.

Reply:
I think this one can be `protected` as well.

Reply:
It will be.